lib/app.rb

```diff
@@ -50,9 +49,10 @@ class App < Sinatra::Base
     use Routes::Comments
     use Routes::Nits
     use Routes::Teams
+    use Routes::Sessions
```

Comment:
But it's not actually checking--it's just defining the checks. So it could technically be first, if you wanted, right?